framework/packages/standards/money-market/src/msg.rs

```diff
@@ -88,23 +89,191 @@ pub enum MoneyMarketQueryMsg {
     },
 
     /// Query using raw asset denoms and addresses
-    #[returns(MoneyMarketQueryResponse)]
-    MoneyMarketRawQuery {
-        /// Actual query
-        query: MoneyMarketRawQuery,
-        /// The name of the dex to interact with
+    /// Deposited funds for lending
+    #[returns(Uint128)]
+    RawUserDeposit {
+        /// User that has deposited some funds
+        user: String,
+        /// Lended asset to query
+        asset: AssetInfoBase<String>,
+        /// Contract Address on which you execute the query
+        contract_addr: String,
+        /// Name of the MoneyMarket to interact with
+        money_market: MoneyMarketName,
+    },
+    #[returns(Uint128)]
+    /// Deposited Collateral funds
+    RawUserCollateral {
+        /// User that has deposited some collateral
+        user: String,
+        /// Collateral asset to query
+        collateral_asset: AssetInfoBase<String>,
+        /// Borrowed asset to query
+        borrowed_asset: AssetInfoBase<String>,
+        /// Contract Address on which you execute the query
+        contract_addr: String,
+        /// Name of the MoneyMarket to interact with
+        money_market: MoneyMarketName,
+    },
+    #[returns(Uint128)]
```

Comment:
Think its bad practice to return raw values here. Better to wrap them in a Response struct.

Reply:
Done